.github/dependabot.yml

```diff
@@ -1,38 +1,25 @@
-
 version: 2
 updates:
-
-  - package-ecosystem: "gitsubmodule"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    target-branch: "develop"
-
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "daily"
     target-branch: "develop"
 
-  - package-ecosystem: "docker"
-    directory: "/build"
-    schedule:
-      interval: "daily"
-    target-branch: "develop"
-
   - package-ecosystem: "gomod"
     directory: "/build"
     schedule:
       interval: "daily"
     target-branch: "develop"
 
-  - package-ecosystem: "maven"
-    directory: "/build"
+  - package-ecosystem: "gomod"
+    directory: "/website"
     schedule:
       interval: "daily"
-    target-branch: "develop"
+    target-branch: "main"
 
```

Comment:
I don't believe that Go is used outside of the website. Can this be reduced to a single entry?

Reply:
We can address this in #422.